src/convertToObject.js

```diff
@@ -5,8 +5,26 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+
+function convertToObject(stylesString) {
+  const styleObject = {};
+
+  const declarations = stylesString
+    .split(';')
+    .filter((declaration) => declaration.trim());
+
+  declarations.forEach((declaration) => {
+    const [property, value] = declaration.split(':');
+
+    if (property && value) {
```

Comment:
The condition `if (property && value)` should be updated to `if (cleanProperty && cleanValue)` to ensure that both `property` and `value` are non-empty after trimming.